scheduler/server/load_based_sched_alg_test.go

```diff
@@ -113,6 +108,40 @@ func Test_Class_Task_Start_Cnts(t *testing.T) {
 			"c6": {loadPct: 3, numRunningTasks: 977, numWaitingTasks: 209145, numJobs: 30, expectedTasksToStart: 0},
 			"c7": {loadPct: 2, numRunningTasks: 2612, numWaitingTasks: 416781, numJobs: 40, expectedTasksToStart: 0}},
 		},
+		// scenario 9 - mimick just diff and tryout running on staging
+		{totalWorkers: 3270, classes: map[string]classState{
+			"land":       {loadPct: 48, numRunningTasks: 0, numWaitingTasks: 0, numJobs: 0, expectedTasksToStart: 0},
```

Comment:
separate from this diff - but I'm realizing now that it seems strange to have all these twitter internal classes in open source. how difficult would it be to pull that out of this repo, use mock classes here, and set the real default percentages in twitter source?

Reply:
I think we can dump this stuff when persistence works

Reply:
I used these tests to validate the loan change scenario. 

We have the default values (land, diff, etc.) coded in load_based_sched_alg.go..... we could change those to a general c0 value that has 100%, and have our twitter code change it to the land, diff, etc values when scheduler starts up, but I thought that was more complicated than needed.  (since we are pretty much the only users of scoot)

Reply:
I disagree with this, unless we scrap this rep altogether this should remain generic / not include company settings.